roles/dtc/remove/tasks/sub_main_external.yml

```diff
@@ -61,12 +61,13 @@
   tags: "{{ nac_tags.remove_links }}"
   when:
     - vars_common_external.changes_detected_fabric_links
-
-- name: Remove Fabric vPC Peering
-  ansible.builtin.import_tasks: common/vpc_peers.yml
-  tags: "{{ nac_tags.remove_vpc_peers }}"
-  when:
-    - vars_common_external.changes_detected_vpc_peering
+    
+# Feature not implemented yet
```

Comment:
Can you expand this comment to indicate that the `vars_common_external.changes_detected_vpc_peering`flag is not defined properly and therefore errors when this task is run?